assets/js/components/ReportTable.stories.js

```diff
@@ -22,26 +22,24 @@
 import ReportTable from './ReportTable';
 import Layout from './layout/Layout';
 import Link from './Link';
-import { CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
 import {
-	createTestRegistry,
-	provideModules,
 	provideModuleRegistrations,
+	provideModules,
 } from '../../../tests/js/utils';
 import WithRegistrySetup from '../../../tests/js/WithRegistrySetup';
 import NewBadge from './NewBadge';
+import { useRegistry } from '../googlesitekit-data';
+import { CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
 
 function Template( args ) {
-	return <ReportTable { ...args } />;
+	const registry = useRegistry();
+	const modules = registry.select( CORE_MODULES ).getModules();
+
+	return <ReportTable { ...args } rows={ Object.values( modules ) } />;
```

Comment:
I'd prefer to keep the template as before. The default rows was in the basic args helper but this should really be specific story rather than the default.